public/less/_repository.less

```diff
@@ -1350,6 +1350,40 @@
             }
         }
     }
+    .segment.reactions, .select-reaction {
+        &.dropdown .menu {
+            right: 0!important;
```

Comment:
I'm not a CSS expert, but `!important` is generally considered a bad practice. Are these necessary?

Reply:
Sadly but it's needed as semantic UI is full of these :(